test/navbar.test.tsx

```diff
@@ -6,16 +6,20 @@ import { Navbar, Button } from '../src'
 
 describe('Navbar', () => {
```

Comment:
A test should be added to make sure that any default prop is getting set correctly. 

There is already a test that makes sure that the button defaults to primary. We could probably make a more general test that tests all default props.

Reply:
Thanks for taking the time to review these changes and I apologize for the little mistakes I feel like such a noob! 😄 
 I'm going to try to slow down a bit and use some other components as reference. I will be resolving these as requested and creating another pull request after double checking everything!

Reply:
Don’t apologize, we’re all learning here 😀! We appreciate your contributions to the project!

Reply:
@ocBruno Just go on, it is fine! We try to enforce some rules for uniformity sake and we are here to review the PR. As @jackcmeyer said, don't apologize! :)